test/runtests.jl

```diff
@@ -7,7 +7,7 @@ using Base.Test
 
 # Conversion and evaluation test
 @test m"expand((1+x)^2)" == m"1 + 2*x + x^2"
-@test mcall(:(exp(im*π))) == -1
```

Comment:
Well, you don't have to make it mandatory. It can still work for other people even if they don't have unicode. However, there is another option: if you use `replace` function in julia, you can replace `π` with `pi` when translating to `MExpr` and then translate `pi` back into `π` without worrying whether the version of maxima supports unicode. That's why I made an extra pair of dictionaries in the `Reduce` package.. because reduce language does not accept unicode. However, I can still translate `π` back and forth from it using `replace`.